app.js

```diff
@@ -5,7 +5,7 @@ const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
 
 // setting up routes
-const index = require('./routes/index');
+// const index = require('./routes/index');
```

Comment:
Hmm we'll still need this for api routes, but the linter won't like that we're including it and not using it...  I guess this is good for now :+1: